.github/workflows/ui.yml

```diff
@@ -57,3 +57,24 @@ jobs:
         run: |
           . venv/bin/activate
           pnpm run test:ct
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+            fetch-depth: 0
+      - name: install gradio
+        run: python -m pip install gradio==$(cat gradio/version.txt)
+      - name: generate theme.css
+        run: python scripts/generate_theme.py
+      - name: install dependencies
+        uses: "./.github/actions/install-frontend-deps"
+        with:
+          always-install-pnpm: true
```

Comment:
I added a new input to this workflow. We can now pass `skip-build: 'true'` here and it will skip building the gradio app since we don't need it in this PR. This should save us a minute or so.

Reply:
scratch this. i missed something.